CommandHelper/LocalPackages/commands/buffs/electro.ms

```diff
@@ -0,0 +1,24 @@
+@plr = player()
+
+
+@hero = associative_array(
+            'name': '§bE§1l§be§1c§bt§1r§bo',
```

Comment:
There are two electros, Jamie Foxx and Francine Frye, which one is this? You will have to update